pydm/widgets/slider.py

```diff
@@ -488,14 +592,29 @@ def value_changed(self, new_val):
         new_val : int or float
             The new value from the channel.
         """
+
+        self.check_if_value_in_map(new_val)
+
+        # Calls find_closest_slider_position_to_value twice.
+        # Comment for the reviewer. I am aware this is not an elegant solution.
```

Comment:
comment on line 599 also can be removed (and it's not performance intensive or happening often or anything so it's fine)